api/schema/schema_validator.go

```diff
@@ -24,11 +24,12 @@ func NewSchemaValidator(t *testing.T, schemaName string, got interface{}) *Schem
 	if err != nil {
 		t.Fatal(err)
 	}
+	// Platform compatibility: use "/" separators always for file://
+	dir = strings.Replace(dir, "\\", "/", -1)
```

Comment:
No, because then it would be pointlessly replacing '/' with '/' on non-Windows platforms ;) This is the only case where it's useful.